### PR TITLE
Merge Oregon State Fair changes

### PR DESCRIPTION
**Pull request by 5t0n3**
Here's a short summary of what was changed:
* Homing and lowering of the new intake was implemented
* We replaced the PCM and some solenoid ports got swapped; these were updated in Constants.java
* A compressor toggle was added to the dashboard to avoid the battery voltage drops it causes, but ended up not being necessary
* Most `SmartDashboard.put*` calls were commented out to avoid cluttering the dashboard
* An RPM offset configurable during the match was added in case we were consistently over/undershooting

I'll probably do some clean-up before finalizing this as well.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -29,10 +31,11 @@
     public static final int kLeftMotorCANId = 13;
     public static final int kRightMotorCANId = 14;
 
-    public static final double kP = 1.2481e-11;
+    public static final double kP = 6e-5; // 1.7637e-8;
     public static final double kI = 0;
+    public static final double kIZone = 100; // rpm
```
Comment (nolanhergert) [inline src/main/java/frc/robot/Constants.java]: This is most useful when kI is non-zero :)

Comment (5t0n3) [reply src/main/java/frc/robot/Constants.java]: Good point, I probably left that in when I was going to add an I which proved to be unnecessary :)

File: src/main/java/frc/robot/commands/utilities/TimeRotation.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
   private double m_power;
   private final Timer m_timer = new Timer();
 
+  // TODO: Rewrite this command to be more accurate
```
Comment (nolanhergert) [inline src/main/java/frc/robot/commands/utilities/TimeRotation.java:13]: What is inaccurate about it?

**Review by nolanhergert — COMMENTED** [review]
Looks good. I think you did the right thing of removing homing and incorporating it into the raising code. It's an implicit assumption that simplifies the code a little.

File: src/main/java/frc/robot/commands/utilities/TimeRotation.java (hunk 2 above)
Comment (5t0n3) [reply src/main/java/frc/robot/commands/utilities/TimeRotation.java:13]: We don't account for the acceleration in the beginning which might throw off the calculated kV value? I'm not entire sure though.

**Comment by 5t0n3** [comment]
I'll probably also test this on the bot tomorrow to make sure I didn't break anything :)

File: src/main/java/frc/robot/commands/utilities/TimeRotation.java (hunk 2 above)
Comment (nolanhergert) [reply src/main/java/frc/robot/commands/utilities/TimeRotation.java:13]: We're talking about a simple TimeRotation command, which uses ArcadeDrive internally...it's not doing Profiled path following / feed forward.

**Comment by 5t0n3** [comment]
Everything seems to be working (I didn't test climb but I didn't make any changes to it beyond during competition). Merging it now :)

**Pull request merged**